library/core/src/slice/mod.rs

```diff
@@ -2599,6 +2599,32 @@ impl<T> [T] {
         }
     }
 
+    /// Fills `self` with elements returned by calling a closure repeatedly.
+    ///
+    /// This method uses a closure to create new values on every push. If
```

Comment:
> push

The word 'push' worked for Vec::resize, but seems a bit out of place for overwriting elements in a slice.

Reply:
Good point; updated!

Reply:
Good point; updated!